tasks/create-cert-webroot.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Check if certificates exists. | Webroot
+  stat:
+    path: /etc/letsencrypt/live/{{ item }}/cert.pem
+  register: _letsencrypt_certs
+  with_items: "{{ certbot_domains }}"
+
+- name: Generate new certificates if they do not exist. | Webroot
+  command: >-
```

Comment:
This command should be run by certbot_user.

otherwise, if it is run as root here, and a renewal is run as cerbot_user, the renewal will fail with permission issue.

Reply:
@timothylytle Could you please review my comment above?

Reply:
Sorry, I had put a comment futher down on the PR,  This was corrected on 2/27 with this (commit)[https://github.com/miarec/ansible-role-letsencrypt/pull/1/commits/957bee4d76c6aa82cc0a3cb015a8d4a69a7da5ad]

I added the following to `tasks/main.yml`
```yaml
- import_tasks: create-cert-webroot.yml
  become: true
  become_user: "{{ certbot_user }}"
  when: certbot_method == 'webroot'
```